.cirrus.yml

```diff
@@ -25,3 +25,19 @@ build_android_task:
     ./flutter/tools/gn --android --unoptimized
     ninja -C out/android_debug_unopt
 
+old_travis_task:
```

Comment:
Can we call this something else?  It won't mean a whole lot in a couple of years when we haven't been using Travis for a long time...

Reply:
Done. Thanks for the feedback! I also planned to reorganize those scripts inside travis/ folder and eventually remove or rename that directory. Hence part of the the reason (other than being lazy) to name it "old_travis" is to remind me to do such reorganization and eventually rename the task to something more meaningful :)

Reply:
Good catch, @gspencergoog , thanks! I copied it from my other scripts which requires `gclient sync` because the content of engine/src/flutter is changed. Nothing changes here so there's no need to do `gclient sync`. Removed.

Reply:
...wrong thread.